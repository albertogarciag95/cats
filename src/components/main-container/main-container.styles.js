import styled from 'styled-components';
import background from "../../assets/background.jpg";

export const CatsContainer = styled.div`
  margin: 0 auto;
  background-image: url(${background});
  text-align: center;
  position: relative;
`;

export const ShuffleButton = styled.div`
  font-size: 2em;
  border-radius: 30px;
  background-color: orange;
  font-family: 'Chango', cursive;
  margin: 2em;
`;

export const CatsContentWrapper = styled.div`
  width: 70%;
  background-color: blue;
  margin: 0 auto;
`;

export const CatsTitle = styled.div`
  font-size: 3em;
  color: #0ea5c8;
  font-family: 'Chango', cursive;
  padding: .5em 0;
`;