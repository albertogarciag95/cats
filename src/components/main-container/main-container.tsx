import React from 'react';
import { CatsContainer, CatsTitle } from './main-container.styles';
import { CatsContent } from '../cats-content/cats-content';

export const MainContainer = () => (
  <CatsContainer>
    <CatsTitle>Random Cat stuff</CatsTitle>
    <CatsContent />
  </CatsContainer>
);