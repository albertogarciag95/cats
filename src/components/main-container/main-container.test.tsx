import renderer from 'react-test-renderer';
import { MainContainer } from './main-container';

test('MainContainer match snapshot', () => {
  const component = renderer.create(
    <MainContainer />
  );
  let cardlink = component.toJSON();
  expect(cardlink).toMatchSnapshot();
});
