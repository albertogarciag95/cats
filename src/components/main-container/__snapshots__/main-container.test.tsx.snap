// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer match snapshot 1`] = `
<div
  className="sc-bdnxRM gZRtiH"
>
  <div
    className="sc-gtsrHT dHFJtM"
  >
    Random Cat stuff
  </div>
  <div
    className="sc-iqAclL bEtPS"
  >
    <div
      className="sc-jrsJWt fpDAci"
    />
    <button
      className="sc-kEqXSa kWxBhd"
      onClick={[Function]}
    >
      Shuffle
    </button>
  </div>
</div>
`;
